pvlib/pvsystem.py

```diff
@@ -1372,6 +1398,67 @@ def get_iam(self, aoi, iam_model='physical'):
         else:
             raise ValueError(model + ' is not a valid IAM model')
 
+    def dc_ohms_from_percent(self):
+        """
+        Calculates the equivalent resistance of the wires using
+        :py:func:`pvlib.pvsystem.dc_ohms_from_percent`
+
+        Makes use of array module parameters according to the
+        following DC models:
+        CEC:
+            `self.module_parameters["V_mp_ref"]`,
+            `self.module_parameters["I_mp_ref"]`,
+        SAPM:
+            `self.module_parameters["Vmpo"]`,
+            `self.module_parameters["Impo"]`,
+        PVsyst-like or other:
+            `self.module_parameters["Vmpp"]`,
+            `self.module_parameters["Impp"]`,
+
+        Other array parameters that are used are:
+        `self.losses_parameters["dc_ohmic_percent"]`,
+        `self.modules_per_string`, and
+        `self.strings`.
+
+        See :py:func:`pvlib.pvsystem.dc_ohms_from_percent` for more details.
+        """
+
+        kwargs = _build_kwargs(['dc_ohmic_percent'],
+                               self.array_losses_parameters)
```

Comment:
Can you change this to `dc_ohmic_percent = self.array_losses_parameters['dc_ohmic_percent']`? Or maybe `dc_ohmic_percent = self.array_losses_parameters.get('dc_ohmic_percent', 0)` if you want to provide a default value. 

The issue is that we have to have a value for `dc_ohmic_percent`, but `_build_kwargs` will return an empty dictionary if it doesn't find the key in the dictionary. That's helpful in other methods that we use `_build_kwargs` so that we avoid needing to provide default values to functions, but it's not what we want here. Sorry it's not clear in the other usages of `_build_kwargs`.